Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -281,12 +285,48 @@ theorem degrees_esymm [Nontrivial R] (n : ℕ) (hpos : 0 < n) (hn : n ≤ Fintyp
 
 end ElementarySymmetric
 
+section CompleteHomogeneousSymmetric
+
+open Classical Set
+
+/-- The `n`th complete homogeneous symmetric `MvPolynomial σ R`. -/
+def hsymm (n : ℕ) : MvPolynomial σ R := ∑ s : Sym σ n, (s.1.map X).prod
+
+lemma hsum_def (n : ℕ) : hsymm σ R n = ∑ s : Sym σ n, (s.1.map X).prod := rfl
+
+@[simp]
+theorem hsymm_zero : hsymm σ R 0 = 1 := by
```

Comment:
You may want to open more namespace to keep things more readable, like `Sym` and `Finset`.

Reply:
Done.